cfme/automate/explorer/instance.py

```diff
@@ -43,19 +41,16 @@ def is_displayed(self):
 class InstanceDetailsView(AutomateExplorerView):
     title = Text('#explorer_title_text')
     table = Table('#instance_fields_grid')
+    domain_priority = Text('//*[@id="instances"]/table/tbody')
 
     @property
     def is_displayed(self):
         return (
             self.in_explorer and
             self.title.text.startswith('Automate Instance [{}'.format(
                 self.context['object'].display_name or self.context['object'].name)) and
-            self.datastore.is_opened and (
-                BZ(1611969, forced_streams=['5.10']).blocks or
-                check_tree_path(
-                    self.datastore.tree.currently_selected,
-                    self.context['object'].tree_path)
-            )
+            self.datastore.is_opened and check_tree_path(self.datastore.tree.currently_selected,
```

Comment:
Optional:
can you please check indentation? but pre-commit or black

Reply:
I checked locally but this bock of code not giving me any error for ```pre-commit run --all```, ```black --check``` or ```flake8```. Hence keeping as it is.